src/Header.js

```diff
@@ -3,12 +3,17 @@ import './Header.css';
 import { NavLink } from 'react-router-dom';
 import Dropdown from 'react-bootstrap/Dropdown';
 import DropdownButton from 'react-bootstrap/DropdownButton';
+import Giphy from './giphy.gif';
```

Comment:
This was a great learning experience.  I did not know how to import images using react before this project.